scripts/main.js

```diff
@@ -1,159 +1,170 @@
-      studentsOnDom(hufflepuff)
-    } else if (e.target.id === "Ravenclaw") {
-      const ravenclaw = students.filter(taco => taco.school === "Ravenclaw" )
-      studentsOnDom(ravenclaw)
-    } else if (e.target.id === "Slytherin") {
-      const slytherin = students.filter(taco => taco.school === "Slytherin" )
-      studentsOnDom(slytherin)
+  renderToDom("#studentContainer", domString)
+  };
+  
+  const voldOnDom = (array) => {
+    let domString = "";
+    const student = array.filter(taco => taco.darkside === false);
+    for (const item of student) {
+    domString += `
+    <div class="card2 rcorners" style="width: 18rem;">
+    <img src="https://rb.gy/mn64l0" class="card-img-top rcorners" alt="...">
+    <div class="card-body" id="studentCard">
+      <h5 class="card-title">${item.name} has joined</h5>
+      <p class="card-text">Voldermort's Army</p>
+    </div>
+  </div>`
+  };
+  renderToDom("#voldContainer", domString)
+  };
+  
+  /* ADD STUDENTS */
+  function addStudent(){ 
+    const input = document.querySelector("#studentInput").value
+    if (input === "") {
+      alert("Please enter your name!");
+    } else {
+    
+    const randomSchool = choices[Math.floor(Math.random()*choices.length)];
+    const newStudent = {
+      id: students.length + 1,
+      image: "",
+      name: input,
+      school: randomSchool,
+      darkside: false
+    };
+    
+    if (randomSchool === "Gryffindor") {
+      newStudent.image = gryffindorImage;
+    } else if (randomSchool === "Ravenclaw") {
+      newStudent.image = ravenclawImage;
+    } else if (randomSchool === "Slytherin") {
+      newStudent.image = slytherinImage;
+    } else {
+      newStudent.image = hufflepuffImage;
     }
     
-});
-
-
-/** EXPEL TO VOLDEMORT ARMY **/
-document.querySelector("#studentContainer").addEventListener("submit",(e) => {
-  if (e.target.id) {
-    const [method, id] = e.target.id.split("--");
-    const index = students.findIndex(taco => taco.id === parseInt(id));
-  if (e.target.id.includes("expel"))
-  badStudents.push(...students.splice(index, 1));
-  studentsOnDom(students);
-  voldOnDom(badStudents);
+    
+    students.push(newStudent);
+    document.querySelector("#studentInput").value="";
+    studentsOnDom(students);
+    }
   };
- });
- };
-
+  
+  
+  /* FILTER BUTTONS */
+    const filterButtons = () => {
+      const domString = `
+      <div class="navigation-btn">
+    <button id="all">All</button>
+    <button id="Gryffindor">Gryffindor</button>
+    <button id="Hufflepuff">Hufflepuff</button>
+    <button id="Ravenclaw">Ravenclaw</button>
+    <button id="Slytherin">Slytherin</button>
+      </div>
+      `;
+      renderToDom('#filterStudents', domString);
+    };
+  
+  
+  /****** EVENT LISTENERS ******/
+  const eventListeners = () => {
+    document.querySelector("#filterStudents").addEventListener("click", (e) => {
+    if (e.target.id === "all") {
```

Comment:
I would use a switch case for this filter instead so I have more than 2 options.